nurbs.js

```diff
@@ -235,14 +249,6 @@ function parseNURBS (points, degree, knots, weights, boundary, opts) {
     this.evaluator = function (derivativeOrder, isBasis) {
       return createEvaluator(this.__cacheKey, this, accessors, this.debug, this.checkBounds, isBasis, derivativeOrder);
     };
-
```

Comment:
Cleanup. These are both obsolete and can be created via `evaluator`. This just reduces unnecessary overhead so that you only compile these functions if you need them.